tools/src/main/scala/scala/scalanative/profiling/Events.scala

```diff
@@ -0,0 +1,16 @@
+package scala.scalanative
+package profiling
+
+sealed trait Event
+object Event {
+  case class Call(ptr: Long, argc: Int)            extends Event
```

Comment:
It might be a better idea to inject those in the beginning of the first basic block, and persist method id instead of the pointer.